app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -0,0 +1,39 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+
+const createSchema = (fields, promise, edit, loadSchema) => ({
+  fields: [
+    {
+      component: componentTypes.SELECT,
+      name: 'ext_management_system_id',
+      id: 'ext_management_system_id',
+      label: __('Storage Manager'),
+      placeholder: __('<Choose>'),
+      onChange: value => promise.then(loadSchema(value)),
```

Comment:
The request will be something like:
```
API.options(`/api/cloud_volumes?ems_id=${value}`).then(loadSchema)),
```
It's [not merged](https://github.com/ManageIQ/manageiq-api/pull/914) yet, so the `ems_id` is not final, but it will return you a schema for sure.